### PR TITLE
Add ability to set TraceWriter/TraceLevel on connection before starting

### PR DESCRIPTION
**Pull request by cwensley**

### CODE REVIEW

File: JabbR.Client/JabbRClient.cs [hunk 1]
```diff
@@ -102,6 +105,28 @@ public ICredentials Credentials
             }
         }
 
+        public TextWriter TraceWriter
+        {
+            get { return _connection != null ? _connection.TraceWriter : _traceWriter; }
```
Comment (davidfowl) [inline JabbR.Client/JabbRClient.cs]: Why not just use auto properties?

Comment (cwensley) [inline JabbR.Client/JabbRClient.cs]: So one can set the tracelevel/writer after the connection is created

Comment (davidfowl) [inline JabbR.Client/JabbRClient.cs]: I dont think it matters anyway

Comment (cwensley) [inline JabbR.Client/JabbRClient.cs]: I suppose.. most of jabbr.client needs to be setup before connecting (events, etc).  Not that I like that it does that.. (;

**Pull request merged**